test-case/check-playback.sh

```diff
@@ -65,6 +68,9 @@ func_lib_setup_kernel_last_line
 func_lib_check_sudo
 func_pipeline_export $tplg "type:playback"
 
+sleep_lst=(0.5 1.9 2 2.1 2.9 3 3.1 5) #And it is 3s for SOF PCI device, 2s for HDA codec in future (not merged).
+j=0
```

Comment:
There are 4 nested `for` loops. This new index is used only in the most inner loop but initialized outside the most outer loop, this is unusual. I think I guessed why but a comment explaining how the sleep_lst iteration works would be useful. A longer variable name may help too.